database/gormdb_assignment_test.go

```diff
@@ -204,16 +204,7 @@ func TestGetAssignmentsWithSubmissions(t *testing.T) {
 		Score:        45,
 		Reviews: []*pb.Review{
 			{
-				ReviewerID: 1, Review: "LGTM!", Feedback: "SGTM!", Score: 42, Ready: true,
-				// TODO(meling) Remove Benchmarks, since benchmarks should only be kept in one place and associated with the assignment (see also TODO in ag.proto)
-				// Benchmarks: []*pb.GradingBenchmark{
-				// 	{
-				// 		Heading: "Ding Dong", Comment: "Communication",
-				// 		Criteria: []*pb.GradingCriterion{
-				// 			{Points: 50, Description: "Loads of ding"},
-				// 		},
-				// 	},
-				// },
+				ReviewerID: 1, Feedback: "SGTM!", Score: 42, Ready: true,
```

Comment:
Should the `Benchmarks` field be re-added now? And I think we should remove the TODO below.

Reply:
Oh, right. Yes, adding it back. Test still passes with it.

Reply:
Done.